keps/sig-node/compute-device-assignment.md

```diff
@@ -87,12 +94,24 @@ message PodResources {
 message ContainerResources {
     string name = 1;
     repeated ContainerDevices devices = 2;
+    repeated int64 cpu_ids = 3;
```

Comment:
do we want an explicit binding between `cpu_id` and the `Topology` information, like we have for `ContainerDevices` below?

Reply:
Given a `cpu_id`, we can obtain `Topology` info so don't think we need it. Is there a specific reason you think we should have this explicitly?

Reply:
I was thinking about consistency, but we can indeed learn this information from `cadvisor` for example, so yes, likely not needed.

Reply:
please account for memory manager kep as well as both work is in parallel.

Reply:
ok, we will describe how it works with MemoryManager.